RecursionLengthOfAString.rb

```diff
@@ -0,0 +1,33 @@
+=begin 
+Write a function that returns the length of a string. Make your function recursive.
+
+Examples
+
+length("apple") ➞ 5
+
+length("make") ➞ 4
+
+length("a") ➞ 1
+
+length("") ➞ 0
+Notes
+Check the Resources tab for info on recursion.
+
+=end
+
+def length(string)
```

Comment:
not recursive. the important thing here is to write a recursive method that would do this.